integration/clean/clean_ami.go

```diff
@@ -0,0 +1,86 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+// +build clean
+
+package main
+
+import (
+	"context"
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/ec2"
+	"github.com/aws/aws-sdk-go-v2/service/ec2/types"
+	smithyTime "github.com/aws/smithy-go/time"
+	"log"
+	"time"
+)
+
+func main() {
+	err := cleanAMI()
+	if err != nil {
+		log.Fatalf("errors cleaning %v", err)
+	}
+}
+
+const daysToKeep = 60
+const keepDuration = -1 * time.Hour * 24 * time.Duration(daysToKeep)
+var expirationDate = time.Now().UTC().Add(keepDuration)
+
+func cleanAMI() []error {
+	log.Print("Begin to clean EC2 AMI")
+
+	cxt := context.Background()
+	defaultConfig, err := config.LoadDefaultConfig(cxt)
+	if err != nil {
+		return []error{err}
+	}
+	ec2client := ec2.NewFromConfig(defaultConfig)
+
+	// Get list of ami
+	nameFilter := types.Filter{Name: aws.String("name"), Values: []string{
+		"cloudwatch-agent-integration-test-ubuntu*",
+		"cloudwatch-agent-integration-test-al2*",
+		"cloudwatch-agent-integration-test-rhel8-base*",
+		"cloudwatch-agent-integration-test-rhel8-1*",
+		"cloudwatch-agent-integration-test-rhel8-2*",
+		"cloudwatch-agent-integration-test-rhel8-3*",
+		"cloudwatch-agent-integration-test-rhel8-4*",
+		"cloudwatch-agent-integration-test-ol8-1*",
+		"cloudwatch-agent-integration-test-ol8-2*",
+		"cloudwatch-agent-integration-test-ol8-3*",
+		"cloudwatch-agent-integration-test-ol8-4*",
```

Comment:
Since all the ami's share the `cloudwatch-agent-integration-test`, would it make since we save this into a const string and use it to make it less wordy and easier to extend the list?